assets/scripts/utils/array.js

```diff
@@ -1,101 +1,124 @@
-import { isArray } from './is';
+import { isArray } from './is'
```

Comment:
This should be replaced with `Array.isArray()`.

Reply:
Import to delete, it is not used within the file.
The `isArray` function probably too as we have the ES6 syntax